### PR TITLE
fix: wrong dirt message in cloud connection

### PR DESCRIPTION
**Pull request by MMaiero**
Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

### CODE REVIEW

**Comment by MMaiero** [comment]
Closes #3994

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 /*******************************************************************************
  * Copyright (c) 2016, 2020 Eurotech and/or its affiliates and others
```
Comment (salvatore-coppola) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: Copyright year is not right.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 import java.util.ArrayList;
 import java.util.Comparator;
+import java.util.logging.Logger;
```
Comment (salvatore-coppola) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: This should be `org.slf4j.Logger`

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java [hunk 3]
```diff
@@ -38,6 +39,8 @@
 
 public class CloudConnectionConfigurationsUi extends Composite {
 
+    protected static final Logger logger = Logger.getLogger(CloudConnectionConfigurationsUi.class.getSimpleName());
```
Comment (salvatore-coppola) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: This should use `org.slf4j.LoggerFactory`

**Review by salvatore-coppola — COMMENTED** [review]
minor problems

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java (hunk 3 above)
Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: I think in the get part the slf4j does not work. But this is a leftover that we can remove.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java (hunk 1 above)
Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: Good catch! I'll make the change

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java (hunk 3 above)
Comment (salvatore-coppola) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudConnectionConfigurationsUi.java]: You are right, i forgot we are on GWT.

**Comment by MMaiero** [comment]
if the changes are ok, can I have your approval so that we can merge?
Thanks

**Pull request merged**